public/iframes/persona-ad.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
```

Comment:
`ads.templewallet.com` should already be able to display Persona ads. Please consider modifying https://github.com/madfish-solutions/templewallet-ads according to your needs and using that site instead.